client/my-sites/activity/activity-log/style.scss

```diff
@@ -69,3 +83,13 @@
 	height: 15%;
 	pointer-events: none;
 }
+
+.activity-log__filterbar-ctn.is-sticky {
+	@include breakpoint-deprecated( '<480px' ) {
+		position: fixed;
+		top: var( --masterbar-height );
+		left: 0;
+		right: 0;
+		z-index: z-index( 'root', '.masterbar' );
```

Comment:
TIL. I didn't know there was a `z-index` function one could you use to set the `z-index` property. Where does it come from? Is it part of the CSS standard?

Reply:
It's a Sass function that has been implemented for Calypso. Z-indices are stored in a map in this file `client/assets/stylesheets/shared/functions/_z-index.scss`, while help avoid using random values.

Reply:
That's super interesting. I wouldn't like to maintain that map, though 😅

Reply:
It's like playing Jenga :)